app/src/main/res/layout/home_fragment.xml

```diff
@@ -15,23 +15,23 @@
             style="?android:attr/progressBarStyle"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:id="@+id/progressBar"
+            android:id="@+id/progress_bar"
             app:layout_constraintEnd_toEndOf="parent"
             android:layout_marginEnd="8dp"
             app:layout_constraintStart_toStartOf="parent"
             android:layout_marginStart="8dp"
             android:visibility="invisible"
             app:layout_constraintHorizontal_bias="0.5"
-            app:layout_constraintTop_toBottomOf="@+id/openButton"
+            app:layout_constraintTop_toBottomOf="@+id/open_button"
             app:layout_constraintBottom_toTopOf="@+id/button"/>
 
     <EditText
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:inputType="textPersonName"
             android:ems="10"
-            android:id="@+id/editText"
-            app:layout_constraintBottom_toTopOf="@+id/openButton"
+            android:id="@+id/edit_text"
```

Comment:
Could we use a more descriptive I'd here?

Reply:
It's only there for testing right now. It will be removed once the search feature is able to handle the searching for stops.

Reply:
I mean it could still use a more descriptive name but whatever